source/material_model/visco_plastic.cc

```diff
@@ -1140,7 +1142,11 @@ namespace aspect
 
           // Include an adiabat temperature gradient in flow laws
           adiabatic_temperature_gradient_for_viscosity = prm.get_double("Adiabat temperature gradient for viscosity");
-
+          if (this->get_heating_model_manager().adiabatic_heating_enabled())
+              AssertThrow (adiabatic_temperature_gradient_for_viscosity == 0.0,
+              ExcMessage("If adiabatic heating is enabled you should not add another adiabatic gradient 
+                         to the temperature for computing the viscosity, because the ambient 
+                         temperature profile already includes the adiabatic gradient.")
```

Comment:
It looks like there is a missing close parenthesis and semicolon ( `);` ) at the end of this statement.

Reply:
The compiler error is also related to missing quotation marks. The code should read as follows:
```
          if (this->get_heating_model_manager().adiabatic_heating_enabled())
               AssertThrow (adiabatic_temperature_gradient_for_viscosity == 0.0,
               ExcMessage("If adiabatic heating is enabled you should not add another adiabatic gradient" 
                          "to the temperature for computing the viscosity, because the ambient" 
                          "temperature profile already includes the adiabatic gradient.")
```